etc/generate-docs.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# usage: ./etc/generate-docs.sh [new version string]
+
+# exit if any command fails
+set -e
+# pipe should fail if any command fails
+set -o pipefail
+
+if ! command -v jazzy > /dev/null; then
+  gem install jazzy || { echo "ERROR: Failed to locate or install jazzy; please install yourself with 'gem install jazzy' (you may need to use sudo)"; exit 1; }
+fi
+
+if ! command -v sourcekitten > /dev/null; then
+  echo "ERROR: Failed to locate SourceKitten; please install yourself and/or add to your \$PATH"; exit 1
+fi
+
+version=${1}
+
+# Ensure version is non-empty
+[ ! -z "${version}" ] || { echo "ERROR: Missing version string"; exit 1; }
+
+# ensure we have fresh build data for the docs generation process
+rm -rf .build
+
+# obtain BSON library and driver versions from Package.resolved
+# write to files first so this script will exit if getting either version fails
+echo "Getting dependency versions..."
+python3 etc/get_dep_version.py swift-son > BSON_VERSION || { echo "Failed to get BSON library version"; rm BSON_VERSION; exit 1; }
```

Comment:
```suggestion
python3 etc/get_dep_version.py swift-bson > BSON_VERSION || { echo "Failed to get BSON library version"; rm BSON_VERSION; exit 1; }
```

Reply:
whoops good catch! I changed that to verify my error handling worked and forgot to switch it back 🙂